src/cf-expandables/usage.md

```diff
@@ -164,15 +169,16 @@ Allows you to float information left and right.
 ## Recommended expandable pattern
 
 Expandables can be built by combining the basic barebones structure described
-in the next section along with more specialized expandable elements
-and modifiers described throughout.
+in the next section along with more specialized expandable elements and
+modifiers described throughout.
 
 ### Default state
 
 The following combination is our recommended go-to expandable pattern.
 
 <div class="o-expandable o-expandable__padded">
-    <button class="o-expandable_header o-expandable_target" title="Expand content">
+    <button class="o-expandable_header o-expandable_target"
+            title="Expand content">
```

Comment:
They're in the [sandbox as well](https://github.com/cfpb/capital-framework-sandbox/blob/gh-pages/components/cf-expandables/index.md). I wasn't sure if @mistergone kept them just because they were there or had a use for them. I'm all for removing them if not.